src/js/components/tablero/UnarchiveIndicators.jsx

```diff
@@ -16,7 +16,45 @@ const Numbers = () => {
   );
 };
 
-const ArchivedIndicators = (props) => {
+
+const PreviewIndicator = props => (
+  <li className="unarchivedItem">
+    <div className="archived-indicator">
+      <div className="row">
+        <div className="col col-3 graph-preview">{props.children}</div>
+        <div className="col col-6">
+          <span>{_.truncate(props.title, { length: 25, omission: '...' })}</span>
+        </div>
+        <div className="col col-3">
+          <span
+            role="button"
+            tabIndex={0}
+            className="unarchive-button"
+            onClick={() => props.handleAdd(props.index, props.type)}
+            onKeyDown={() => props.handleAdd(props.index, props.type)}
+          >
+            Unarchive
```

Comment:
Do you support localization? If yes, then you should use translation here

Reply:
At the moment we don't have localization